theano/gpuarray/dnn_gi.c

```diff
@@ -13,8 +13,8 @@ if (PARAMS->choose_algo) {
 
 int reuse_algo;
 cudnnConvolutionBwdDataAlgo_t prev_algo;
-size_t prev_kern_dims[5] = {0};
-size_t prev_top_dims[5] = {0};
+size_t prev_kern_dims[5];
+size_t prev_top_dims[5];
```

Comment:
It is indeed done ! Here: https://github.com/notoraptor/Theano/blob/ee22927e1e74001df7b115339ab1d5e482b6fb23/theano/gpuarray/dnn_gi.c#L6